README.md

```diff
@@ -15,7 +15,7 @@ Additionally, we identified morphological features that were critical for the mo
 These findings suggest that specific cellular structures are strong indicators of heart cell health or failure.
 The processing methods and results of this analysis can be accessed [here](https://github.com/WayScience/cellpainting_predicts_cardiac_fibrosis).
 
-Building upon these results, we advanced our study to the next level by leveraging high-content screening (HCS) with CP assay. 
+Building upon these results, we advanced our study to the next level by leveraging high-content screening (HCS) with CP assay.
```

Comment:
Is there a relevant link to include for `high-content screening (HCS) with CP assay`? If so, consider adding it as a reference. It could also help to consider adding a `CITATION.cff` file where you could store various related references for the project.